variants/tlora_t3s3_epaper/platformio.ini

```diff
@@ -14,10 +14,10 @@ build_flags =
   -DEINK_LIMIT_FASTREFRESH=10          ; How many consecutive fast-refreshes are permitted
   -DEINK_LIMIT_RATE_BACKGROUND_SEC=30  ; Minimum interval between BACKGROUND updates
   -DEINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
-;   -D EINK_LIMIT_GHOSTING_PX=2000      ; (Optional) How much image ghosting is tolerated
-  -DEINK_BACKGROUND_USES_FAST          ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
-  -DEINK_HASQUIRK_GHOSTING             ; Display model is identified as "prone to ghosting"
-  -DEINK_HASQUIRK_WEAKFASTREFRESH      ; Pixels set with fast-refresh are easy to clear, disrupted by sunlight
+  -DEINK_HASQUIRK_VICIOUSFASTREFRESH   ; Identify that pixels drawn by fast-refresh are harder to clear
+  ;-DEINK_LIMIT_GHOSTING_PX=2000        ; (Optional) How much image ghosting is tolerated
+  ;-DEINK_BACKGROUND_USES_FAST          ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
```

Comment:
I'm not sure it'd handle constantly doing 10 fast refreshes back-to-back with the current driver, but now that `EINK_BACKGROUND_USES_FAST` is no-longer defined, that sort of back-to-back fast refresh will be much rarer; so the display can probably tolerate it.

Reply:
You mean that not optimal values can physically damage the display? Or is it "aging" faster? Did you ever manage to "break" an e-ink using wrong settings?

Reply:
> You mean that not optimal values can physically damage the display?

If left unchecked yes, although in my experience, the image quality significantly deteriorates before damage is irreversible.

> Or is it "aging" faster?

Quite possibly, although unless something is set *really* wrong, I doubt it's too much of an issue.

> Did you ever manage to "break" an e-ink using wrong settings?

I've never had one pushed to breaking point before, although I've run them down into some pretty rough states. If things are going wrong, it tends to become visible quite quickly.

> I've observed the display now for a couple of hours. I don't see something suspicious.

A couple of hours would probably be long enough for any problems to start to show, so it's a good sign.

> Every now and then it is doing a full refresh even without pressing a button (probably due to EINK_HASQUIRK_VICIOUSFASTREFRESH).

That's the behavior when `EINK_BACKGROUND_USES_FAST` is not defined. If an urgent display update happens, like in response to button press, fast-refresh is used, however the display will redraw the same screen image ~30 seconds later using a full refresh. Any non-urgent updates will also use full-refresh.

This configuration in particular is what's protecting this display right now. The frequent full-refreshing during downtime is helping prevent charge imbalances accumulating from this displays fast-refresh operation, which is currently quite aggressive. 

There's a flowchart over on #3316 if you ever have to try make sense of my code spaghetti there.